metricbeat/mb/testing/data_generator.go

```diff
@@ -50,7 +50,7 @@ func WriteEvent(f mb.EventFetcher, t testing.TB) error {
 	}
 
 	fullEvent := CreateFullEvent(f, event)
-	WriteEventToDataJSON(t, fullEvent, "")
+	WriteEventToDataJSON(t, fullEvent, ".")
```

Comment:
@jsoriano Not sure if I have to make this change because of https://github.com/elastic/beats/pull/10367 Seems to have worked a few days ago without the dot. No deal breaker.

Reply:
Yes, this is related for sure :slightly_frowning_face: I'll make `""` to behave as `"."` for BC.

Reply:
https://github.com/elastic/beats/pull/10402